examples/empty/project/Dependencies.scala

```diff
@@ -1,17 +1,18 @@
 import sbt._
 
 object Dependencies {
+  val scope = Provided
```

Comment:
Not sure extracting it to a val changes anything.

Reply:
True, I'm going to revert it or change to the `Runtime` scope. I was experimenting with different scopes, as I had trouble with `Provided` when running the `InputGenerator`.